Library/Homebrew/test/.rubocop.yml

```diff
@@ -2,7 +2,6 @@ inherit_from:
   - ../.rubocop.yml
   - .rubocop_todo.yml
 
-AllCops:
-  RSpec:
-    Patterns:
-    - ./*
+RSpec:
+  Include:
+  - ./*
```

Comment:
Shouldn't this be `**/*.rb`?

Reply:
I dunno, it's the same glob as before. The yaml only changed to avoid some warnings in rubocop-rspec when running `brew style`. It doesn't seem to make a difference regardless.